aspnet-core/xml/ns-Microsoft.AspNetCore.SignalR.Client.xml

```diff
@@ -1,6 +1,6 @@
 <Namespace Name="Microsoft.AspNetCore.SignalR.Client">
   <Docs>
-    <summary>To be added.</summary>
-    <remarks>To be added.</remarks>
+    <summary>Provides types that are used when connecting to a server endpoint that implmenets SignalR.</summary>
```

Comment:
Nit: Why "Contains types" in some places and "Provides types" in others? 

```suggestion
    <summary>Contains types that are used for communicating with a SignalR server.</summary>
```

Feel free to change the phrasing, but we should at least fix "implmenets".